src/renderer/features/proxies/model/proxies-model.ts

```diff
@@ -417,4 +417,8 @@ export const proxiesModel = {
     workerStarted,
     proxiedWalletsCreated,
   },
+
+  output: {
+    proxiedWalletsCreationFinished: createProxiedWalletsFx.doneData,
```

Comment:
I would suggest a simpler name `walletsCreated`, we already know the domain - `proxies-model`
no need for such a specific name